packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -371,6 +521,7 @@ function completeWork(
       break;
     }
     case HostRoot: {
+      updateHostContainer(workInProgress);
```

Comment:
Is host environment throws, does it mean we don't clean up the stack due to `pop`s being moved after?

Reply:
I'll revert this, mistake when resolving conflicts